Packs/PrismaCloudCompute/Layouts/layoutscontainer-Prisma_Cloud_Compute_Audit_Layout_v2.json

```diff
@@ -77,11 +77,20 @@
                             {
                                 "endCol": 2,
                                 "fieldId": "externallink",
-                                "height": 53,
+                                "height": 22,
                                 "id": "953e82a0-0bc8-11ed-a263-1f3efd1f9f94",
                                 "index": 6,
                                 "sectionItemType": "field",
                                 "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "itemowneremail",
+                                "height": 22,
+                                "id": "9b8fc660-8c4f-11ee-bbb6-c30980c106c7",
+                                "index": 7,
+                                "sectionItemType": "field",
+                                "startCol": 0
```

Comment:
Consider changing the "Container Alerts" tab to a more general tab that summarize investigation results/findings that contains Container alerts, Malicious Indicators found, Exploitable CVE's or any other findings that will be readable and will allow the user to see a summary and decide on further actions.